lambda/custom/package.json

```diff
@@ -6,12 +6,13 @@
   "scripts": {
     "setup": "bash ../../bin/setup.sh",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "clean": "rimraf .build",
+    "clean": "rimraf .build build.zip ../../.ask/lambda/custom",
```

Comment:
It would be useful to also have a `clean` type command to fully reset the environment (i.e. deleting config files as well). Useful for development, anyway (: